scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -373,11 +390,19 @@ protected static Options getOptions() {
         help.setRequired(false);
         options.addOption(help);
 
-        Option rmURL = new Option("u", "rmURL", true, "the resource manager URL (default: localhost)");
+        Option rmURL = new Option("u", "rmURL", true, "bind to a given resource manager URL (default: localhost)\");");
         rmURL.setArgName("rmURL");
         rmURL.setRequired(false);
         options.addOption(rmURL);
 
+        Option schedulerUrl = new Option("s",
+                                         "scheduler-url",
+                                         true,
+                                         "bind to a given scheduler URL (default: localhost)");
+        schedulerUrl.setArgName("scheduler-url");
+        schedulerUrl.setRequired(false);
+        options.addOption(schedulerUrl);
+
```

Comment:
Is semantic incompatibility handled ?
For example, if scheduler-url is provided, should rm-url be also provided ?

In that case I suppose the command can only start the rest server. Maybe some more detailed description of the parameters would be nice.